beets/importer.py

```diff
@@ -1385,6 +1412,11 @@ def apply_choice(session, task):
 
     task.add(session.lib)
 
+    # If ``set_fields`` is set, set those fields to the
+    # configured values.
+    if config['import']['set_fields']:
+        task.set_fields()
```

Comment:
I actually used to have it structured that way, but it doesn't work. Without the call to `task.add` first, there's no `album` attribute defined on the task: `AttributeError: 'ImportTask' object has no attribute 'album'`.

Please let me know it you have another idea, because I do agree with you it would be better.